logging/structured/src/main/scala/tofu/logging/Logging.scala

```diff
@@ -33,8 +33,8 @@ trait LoggingBase[F[_]] {
     write(level, message, values: _*)
 
   /** could be overridden in the implementations, write message about some exception */
-  def writeCause(level: Level, message: String, cause: Throwable): F[Unit] =
-    write(level, message, Logging.errorvalue(cause))
+  @silent def writeCause(level: Level, message: String, cause: Throwable, values: LoggedValue*): F[Unit] =
+    write(level, message, values: _*)
```

Comment:
I think the previous version could do something strange. So I changed it a bit.

Reply:
Now we lost the `cause`, I suppose appending `values :+ Logging.errorvalue(cause)` would be more suitable here

Reply:
I propose next changes:

1. Move `Logging.errorvalue` to `LoggedValue.error`
2. Change the result of `LoggedValue.error` to the new wrapper
   that is sublclass of both `LoggedValue` and `Throwable(cause.message, cause)`.